### ISSUE TITLE
[plugin] Compiled code uses undefined variable for imported DTO file

### ISSUE THREAD

**Issue by Menci** [nestjs/swagger#422]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
Import another dto class in a dto file and use it as a member:

```ts
// group-meta.dto.ts
export class GroupMetaDto {
  readonly id: number;
  readonly name: string;
  readonly ownerId: number;
}
```

```ts
// get-group-meta-response.dto.ts
import { GroupMetaDto } from "./group-meta.dto";

export enum GetGroupMetaResponseError {
  NO_SUCH_GROUP = "NO_SUCH_GROUP"
}

export class GetGroupMetaResponseDto {
  groupMeta?: GroupMetaDto;
  error?: GetGroupMetaResponseError;
}
```

```json
// tsconfig.json
{
  "compilerOptions": {
    "module": "commonjs",
    "declaration": true,
    "removeComments": true,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "target": "es2017",
    "sourceMap": true,
    "outDir": "./dist",
    "baseUrl": "./",
    "incremental": true,
    "paths": {
      "@/*": ["src/*"]
    }
  },
  "exclude": ["node_modules", "dist"]
}
```

```json
// nest-cli.json
{
  "collection": "@nestjs/schematics",
  "sourceRoot": "src",
  "compilerOptions": {
    "plugins": [
      {
        "name": "@nestjs/swagger/plugin",
        "options": {
          "classValidatorShim": true
        }
      }
    ]
  }
}
```

Result:

```
/root/syzoj-ng/dist/group/dto/get-group-meta-response.dto.js:21
    __metadata("design:type", group_meta_dto_1.GroupMetaDto)
                              ^

ReferenceError: group_meta_dto_1 is not defined
    at Object.<anonymous> (/root/syzoj-ng/dist/group/dto/get-group-meta-response.dto.js:21:31)
    at Module._compile (internal/modules/cjs/loader.js:1121:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1160:10)
    at Module.load (internal/modules/cjs/loader.js:976:32)
    at Function.Module._load (internal/modules/cjs/loader.js:884:14)
    at Module.require (internal/modules/cjs/loader.js:1016:19)
    at require (internal/modules/cjs/helpers.js:69:18)
    at Object.<anonymous> (/root/syzoj-ng/dist/group/dto/index.js:8:10)
    at Module._compile (internal/modules/cjs/loader.js:1121:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1160:10)
```

The compiled code is:

```js
"use strict";
var __decorate = (this && this.__decorate) || function (decorators, target, key, desc) {
    var c = arguments.length, r = c < 3 ? target : desc === null ? desc = Object.getOwnPropertyDescriptor(target, key) : desc, d;
    if (typeof Reflect === "object" && typeof Reflect.decorate === "function") r = Reflect.decorate(decorators, target, key, desc);
    else for (var i = decorators.length - 1; i >= 0; i--) if (d = decorators[i]) r = (c < 3 ? d(r) : c > 3 ? d(target, key, r) : d(target, key)) || r;
    return c > 3 && r && Object.defineProperty(target, key, r), r;
};
var __metadata = (this && this.__metadata) || function (k, v) {
    if (typeof Reflect === "object" && typeof Reflect.metadata === "function") return Reflect.metadata(k, v);
};
Object.defineProperty(exports, "__esModule", { value: true });
const openapi = require("@nestjs/swagger");
var GetGroupMetaResponseError;
(function (GetGroupMetaResponseError) {
    GetGroupMetaResponseError["NO_SUCH_GROUP"] = "NO_SUCH_GROUP";
})(GetGroupMetaResponseError = exports.GetGroupMetaResponseError || (exports.GetGroupMetaResponseError = {}));
class GetGroupMetaResponseDto {
}
__decorate([
    openapi.ApiProperty({ required: false, type: () => require("./group-meta.dto").GroupMetaDto }),
    __metadata("design:type", group_meta_dto_1.GroupMetaDto)
], GetGroupMetaResponseDto.prototype, "groupMeta", void 0);
__decorate([
    openapi.ApiProperty({ required: false }),
    __metadata("design:type", String)
], GetGroupMetaResponseDto.prototype, "error", void 0);
exports.GetGroupMetaResponseDto = GetGroupMetaResponseDto;
//# sourceMappingURL=get-group-meta-response.dto.js.map
```

The plugin generated `group_meta_dto_1` but TypeScript generated `require("./group-meta.dto")` without a variable.

## Expected behavior
It compiles to runable js code.

## Minimal reproduction of the problem with instructions
1. Create a nestjs project with `nest-cli` and latest `@nestjs/swagger` package.
2. Use the `tsconfig.json` and `nest-cli.json` above, add DTOs above to the project.
3. Create a controller, with a method with the result `GetGroupMetaResponseDto`.
4. `yarn start`

## Environment

<pre><code>
Nest version: 
    "@nestjs/common": "^6.7.2",
    "@nestjs/core": "^6.7.2",
    "@nestjs/swagger": "^4.0.9",
 
For Tooling issues:
- Node version: v13.2.0
- Platform:  Linux

Others:
    "typescript": "^3.6.3"
</code></pre>

**Comment by kamilmysliwiec** [comment]
Please, provide a minimal repository which reproduces your issue.

**Comment by Menci** [comment]
https://github.com/Menci/nestjs-swagger-issue-422

**Comment by kamilmysliwiec** [comment]
Quick workaround for now:

Change:
```typescript
import { GroupMetaDto } from "./group-meta.dto";
```
to:
```typescript
import * as groupMetaDto from "./group-meta.dto";
```
and use as follows:
```typescript
groupMeta?: groupMetaDto.GroupMetaDto;
```

**Comment by Menci** [comment]
Thanks. I'll take this workaround and look forward to a fix.

**Comment by cjancsar** [comment]
Running in to same issue, and when using the workaround posted above, the enum values do not appear in swagger docs:
Shows this:
![image](https://user-images.githubusercontent.com/51799534/71022005-79149d00-20cd-11ea-99a0-00d0ec06b03a.png)

Instead of this:
(When manually setting @ApiProperty to `@ApiProperty({ enum: constants.NotificationTypes, default: constants.NotificationTypes.PAYMENT_RECEIPT, isArray: false })`)
![image](https://user-images.githubusercontent.com/51799534/71021895-3f439680-20cd-11ea-9794-09b9f678c2a8.png)

**Comment by kamilmysliwiec** [comment]
@cjancsar @Menci can you test with @next release? `npm i @nestjs/swagger@next`

**Comment by kamilmysliwiec** [comment]
Fixed in 4.1.1. Thanks for reporting!
